sharepoint-adfs/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change log for Azure template SharePoint-ADFS
 
+## Enhancements & bug-fixes - Published in October 5, 2020
+
+* Implement workaround to the template validation error when it is deployed from the portal and parameter numberOfAdditionalFrontEnd is set to 0
```

Comment:
What's the validation error you're seeing?

Reply:
The error in the portal is the following:

Error submitting the deployment request.
Additional details from the underlying API that might be helpful: Deployment template validation failed: 'The template 'copy' definition at line '928' and column '15' has an invalid copy count. The copy count must be a positive integer value and cannot exceed '800'. Please see https://aka.ms/arm-copy for usage details.'.

Reply:
I don't see that error (and should have been fixed quite a while ago) - which portal/cloud do you see it in?

Reply:
I reproduce it each time using the default parameters (especially numberOfAdditionalFrontEnd = 0) when I deploy the template from this link:
https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fsharepoint-adfs%2Fazuredeploy.json

Reply:
ok, looks like portal had a regression (had to rollback the apiVersion)... so it's not a template problem, but a portal problem.  The are rolling out the fix so I'd prefer to wait for that rather than change the template.

That said, if you're getting feedback on the failures and want to push this through, we can, I'd just like a commitment to rollback once portal is updated.

Reply:
This is great news. Please approve the PR, I'm already working on a minor update of the template that I'll submit sometime next week (or the week after in worst case), and I'll very gladly revert the workaround

Reply:
Done.... aside, do you know of any plans to fix the other sharepoint samples in the repo?  Or should we remove them?

Reply:
Thanks. IMO the other SharePoint templates should be deleted, they are not maintained, outdated, and very probably not working for a very long time

Reply:
Do you mind submitting PRs for removing them?

Reply:
Sure, I'll submit 1 for each template today